src/applications/check-in/travel-claim/pages/travel-address/index.jsx

```diff
@@ -24,7 +24,7 @@ const TravelAddress = props => {
     {
       info: (
         <Trans
-          i18nKey="if-you-traveled-from-a-different-address--helptext"
+          i18nKey="if-you-traveled-from-a-different-address-bold--helptext"
```

Comment:
see comment on bold updates in translation file.

Reply:
The address data is now in mock data in `checkInData.veteranData.address`, so that can be added.